cmd/nerdctl/version.go

```diff
@@ -72,6 +72,8 @@ func versionAction(cmd *cobra.Command, args []string) error {
 	} else {
 		fmt.Fprintf(w, "Client:\n")
 		fmt.Fprintf(w, " Version:\t%s\n", v.Client.Version)
+		fmt.Fprintf(w, " Platform:\t%s\n", v.Client.Os)
```

Comment:
We can simple  do like docker  `OS/Arch`

Reply:
Make sense. I will update the code